tests/network/localnet/liblocalnet.py

```diff
@@ -1,15 +1,60 @@
+import tests.network.libs.nodenetworkconfigurationpolicy as libnncp
 from libs.net import netattachdef
-from libs.net.vmspec import add_network_interface, add_volume_disk
+from libs.net.traffic_generator import Client, Server
+from libs.net.vmspec import IP_ADDRESS, add_network_interface, add_volume_disk, lookup_iface_status
 from libs.vm.affinity import new_pod_anti_affinity
 from libs.vm.factory import base_vmspec, fedora_vm
 from libs.vm.spec import CloudInitNoCloud, Interface, Metadata, Multus, Network
 from libs.vm.vm import BaseVirtualMachine, cloudinitdisk_storage
 from tests.network.libs import cloudinit
 
-NETWORK_NAME = "localnet-network"
+_IPERF_SERVER_PORT = 5201
+NNCP_INTERFACE_TYPE_OVS_BRIDGE = "ovs-bridge"
 
 
-def localnet_vm(namespace: str, name: str, network: str, cidr: str) -> BaseVirtualMachine:
+def run_vms(
+    vms: tuple[BaseVirtualMachine, BaseVirtualMachine],
+) -> tuple[BaseVirtualMachine, BaseVirtualMachine]:
+    for vm in vms:
+        vm.start()
+    for vm in vms:
+        vm.wait_for_ready_status(status=True)
+        vm.wait_for_agent_connected()
+    return vms
+
+
+def create_traffic_server(vm: BaseVirtualMachine) -> Server:
+    return Server(vm=vm, port=_IPERF_SERVER_PORT)
+
+
+def create_traffic_client(vms: tuple[BaseVirtualMachine, BaseVirtualMachine], network_name: str) -> Client:
```

Comment:
this is a function (not a fixture) so it should get client and server explicitly and not a tuple